packages/components/src/dropdown-menu/index.js

```diff
@@ -55,27 +63,33 @@ function DropdownMenu( {
 						onKeyDown={ openOnArrowDown }
 						aria-haspopup="true"
 						aria-expanded={ isOpen }
-						label={ label }
-						tooltip={ label }
+						label={ buttonLabel }
+						labelPosition={ labelPosition }
+						tooltip={ buttonLabel }
 					>
-						<span className="components-dropdown-menu__indicator" />
+						{ icon !== 'ellipsis' && <span className="components-dropdown-menu__indicator" /> }
```

Comment:
It feels like `ellipsis` should be the default icon. It doesn't make sense to render an indicator when the ellipsis is selected as an icon. I'm not happy about this solution but otherwise, we will have to hide `components-dropdown-menu__indicator` with CSS in 2 places.

Reply:
> It feels like `ellipsis` should be the default icon. It doesn't make sense to render an indicator when the ellipsis is selected as an icon. I'm not happy about this solution but otherwise, we will have to hide `components-dropdown-menu__indicator` with CSS in 2 places.

From https://github.com/WordPress/gutenberg/pull/14843#discussion_r272653497, I think it could be reasonable with design confirmation that there's a redundancy between the ellipsis and indicator which makes it reasonable to consolidate to one (i.e. hide the indicator by code when the icon is `ellipsis`).

Reply:
From a brief chat with @gziolo, I _think_ the current behavior is right.

Although there doesn't appear to be any super firm guidelines for a canonical "dropdown menu icon", commonly it _is_ the triangle. [Material uses it a lot](https://material.io/design/components/menus.html#usage), at least. The ellipsis, however, is the exception. It should be used when you know what you're doing, because the ellipsis is "overflow", so it's:

- always at the end of a series of buttons
- there's only one in that series of buttons

So it seems like defaulting to the triangle but allowing the user to feed it another icon like the ellipsis, is the right behavior.

Reply:
So, this is generally agreeable to me. But I wonder if we should be considering this in a more formalized way than an ad hoc exception. For example, I could imagine this would be sensible if the _default_ `icon` for this component was `'ellipsis'`, and the separator was included as an expression of "If you're not using the safe default, we'll assume you need the indicator". But the default is currently `'menu'`, which itself is a concern as well, since I think the same justification could be applied to the [menu icon](https://developer.wordpress.org/resource/dashicons/#menu) as not needing the indicator.

A few options I might propose:

- Collect a whitelist of icons we think do not need to have the indicator assigned
- Make the exemption explicit, so that the consumer must define whether it is or isn't to be shown

Reply:
https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/rich-text/format-toolbar/index.js#L23-L28

It looks like we display indicator only when `false` is explicitly set for the `icon` prop. I checked all other occurrences in our code. It probably should be the default icon but as you already pointed out, let's not introduce breaking changes. I will adjust the code to check against falseness of `icon` to move forward and make the implementation simpler.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/14843/commits/032167abd4a4e61be165fee0344e9bfdfdf9499a.